docs/modules/ROOT/pages/index.adoc

```diff
@@ -228,24 +228,25 @@ This specification defines a _<<C2PA assertion>>_ known as an *<<_identity_asser
 
 The *<<_identity_assertion,identity assertion>>* contains the following data fields:
 
-* `referenced_assertions` contains a list of assertions that are referenced and signed by this _<<_credential_subject,credential subject>>_
+* `tbs` (“to be signed”) contains a set of data to be signed by this _<<_credential_subject,credential subject>>_. This structure contains:
+** `referenced_assertions` contains a list of other assertions selected by the credential subject. This is referred to as `tbs.referenced_assertions` throughout this specification.
 * `signature` contains the raw byte stream of the _<<_credential_subject,credential subject>>’s_ signature
 * `sig_type` defines the data type of signature
 * `pad1` and `pad2` are zero-filled byte strings used to fill space
 
-The content of `referenced_assertions` is a list of assertions which is presented to the _<<_credential_subject,credential subject>>_ for signature. This list of _<<_referenced_assertions,referenced assertions>>_ MUST include a _<<hard_binding,hard binding>>_ assertion as described in link:++https://c2pa.org/specifications/specifications/2.0/specs/C2PA_Specification.html#_hard_bindings++[Section 9.2, “Hard bindings,”] of the C2PA technical specification.
+The content of `tbs` is a data structure which is presented to the _<<_credential_subject,credential subject>>_ for signature. The list of _<<_referenced_assertions,referenced assertions>>_ contained within `tbs` MUST include a _<<hard_binding,hard binding>>_ assertion as described in link:++https://c2pa.org/specifications/specifications/2.0/specs/C2PA_Specification.html#_hard_bindings++[Section 9.2, “Hard bindings,”] of the C2PA technical specification.
```

Comment:
how is the information serialized into a "blob" for presentation?  Or is the expectation that the actual JSON/CBOR object will be presented?

Reply:
Adding language that says it's specified per type.

Reply:
Suggest we add similar text to the CC spec 
https://c2pa.org/specifications/specifications/2.0/specs/C2PA_Specification.html#_signing_a_claim

"The canonical serialized CBOR of the tbs map is presented to..."